hivemind/server/layers/__init__.py

```diff
@@ -1,19 +1,15 @@
 import torch
 
+name_to_block = {}
+name_to_input = {}
+
 from hivemind.server.layers.common import FeedforwardBlock, TransformerEncoderLayer, NopExpert
 from hivemind.server.layers.dropout import DeterministicDropout, DeterministicDropoutNetwork
 from hivemind.server.layers.lr_schedule import get_linear_schedule_with_warmup
 
-name_to_block = {'ffn': lambda hid_dim: FeedforwardBlock(hid_dim),
-                 'transformer': lambda hid_dim: TransformerEncoderLayer(hid_dim, dim_feedforward=4 * hid_dim, nhead=16),
-                 'nop': lambda hid_dim: NopExpert(hid_dim),
-                 'det_dropout': lambda hid_dim: DeterministicDropoutNetwork(hid_dim, dropout_prob=0.2)}
+from hivemind.server.layers.custom_experts import *
```

Comment:
No wildcard imports should be used, if possible

Reply:
ref: https://www.python.org/dev/peps/pep-0008/#id23

Reply:
Fixed